cloud/docker/docker.py

```diff
@@ -27,7 +27,9 @@
 version_added: "1.4"
 short_description: manage docker containers
 description:
-  - Manage the life cycle of docker containers.
+  - This is the original Ansible module for managing the Docker container life cycle.
+  - NOTE: Additional and newer modules are available. For the latest on orchestrating containers with Ansible
+    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docsite/rst/guide_docker.rst).
```

Comment:
There's an extra character, a "U", before the URL.  The URL doesn't need parenthesis around it when written this way.